### PR TITLE
Feature/eslint

### PR DESCRIPTION
**Pull request by ANDREYDEN**
Linting potions and styling changes

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -13,12 +13,13 @@ let IS_OUT_FILE_EMPTY = false
...
-	if (type === 'u') params.urls = value
-	if (type === 'o') params.output = value
+	if (type === 'i') { params.input = value }
+	if (type === 'o') { params.output = value }
 })
+
 // exits if no input or output file provided
-params.urls || (function(){console.log('ERROR: missing urls file\n to add use u: parameter.');process.exit(1)}())
-params.output || (function(){console.log('ERROR: missing output file\n to add use o: parameter.');process.exit(1)}())
+params.input || (function () { console.log('ERROR: missing input file\n to add use i: parameter.'); process.exit(1) }())
```
Comment (tymnim) [inline main.js]:
```suggestion
params.urls || (function () { console.log('ERROR: missing urls file\n to add use u: parameter.'); process.exit(1) }())
```

File: main.js [hunk 2]
```diff
@@ -13,12 +13,13 @@ let IS_OUT_FILE_EMPTY = false
 const params = {}
 process.argv.slice(2).forEach(param => {
 	const [type, value] = param.split(':')
-	if (type === 'u') params.urls = value
-	if (type === 'o') params.output = value
+	if (type === 'i') { params.input = value }
```
Comment (tymnim) [inline main.js]:
```suggestion
	if (type === 'u') { params.urls = value }
```

File: main.js [hunk 3]
```diff
@@ -31,20 +32,19 @@ if (isMainThread) {
...
+ * @param input - string, path to the file which contains paths to content files
 /*
  * @param urls {string} - the path to the file containing all the urls to fetch
  * @param output {string} - the path to the file that will contain the resulting objects
  */
-async function main({ urls, output }) {
-
-	if (! fs.existsSync(urls)) {
+async function main({ input, output }) {
+	if (!fs.existsSync(input)) {
```
Comment (tymnim) [inline main.js]:
```suggestion
async function main({ urls, output }) {
	if (!fs.existsSync(urls)) {
```

File: main.js [hunk 4]
```diff
@@ -31,20 +32,19 @@ if (isMainThread) {
  * 	gets file list
  * 	manages workers
  * 	saves result
- * @{
- * 	input - string, path to the file which contains paths to content files
+ * @param input - string, path to the file which contains paths to content files
```
Comment (tymnim) [inline main.js]:
```suggestion
```
I think it should be removed

Comment (ANDREYDEN) [reply main.js]: true

**Pull request merged**